Build failure: pkgsLLVM.libxcrypt

### Steps To Reproduce

Steps to reproduce the behavior:

I have the following flake:

```nix
{
  description = "Clang flake.";

  inputs = {
    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
    flake-utils.url = "github:numtide/flake-utils";
  };
  outputs = { self, nixpkgs, flake-utils, ... }: 
    flake-utils.lib.eachDefaultSystem (system: let
      pkgs = import nixpkgs {
        inherit system;
      };
    in {
      packages.default = pkgs.pkgsLLVM.libxcrypt;
    });
}
```

When I try to build `pkgs.pkgsLLVM.libxcrypt` it fails with the log below. It builds fine with gcc.

### Build log

```
@nix { "action": "setPhase", "phase": "unpackPhase" }
Running phase: unpackPhase
unpacking source archive /nix/store/f76sw6av28v3cv6nxmqw190pkpnfnw9b-libxcrypt-4.4.36.tar.xz
source root is libxcrypt-4.4.36
setting SOURCE_DATE_EPOCH to timestamp 1688579904 of file libxcrypt-4.4.36/config.h.in
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
Updating Autotools / GNU config script to a newer upstream version: ./build-aux/m4-autogen/config.sub
Updating Autotools / GNU config script to a newer upstream version: ./build-aux/m4-autogen/config.guess
@nix { "action": "setPhase", "phase": "configurePhase" }
Running phase: configurePhase
fixing libtool script ./build-aux/m4-autogen/ltmain.sh
./configure
patching script interpreter paths in ./configure
./configure: interpreter directive changed from "#! /bin/sh" to "/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/sh"
configure flags: --disable-static --disable-dependency-tracking --prefix=/nix/store/hmxnjscrn38r6lq91h05vc7vpa9pwqdy-libxcrypt-x86_64-unknown-linux-gnu-4.4.36 --bindir=/nix/store/hmxnjscrn38r6lq91h05vc7vpa9pwqdy-libxcrypt-x86_64-unknown-linux-gnu-4.4.36/bin --sbindir=/nix/store/hmxnjscrn38r6lq91h05vc7vpa9pwqdy-libxcrypt-x86_64-unknown-linux-gnu-4.4.36/sbin --includedir=/nix/store/hmxnjscrn38r6lq91h05vc7vpa9pwqdy-libxcrypt-x86_64-unknown-linux-gnu-4.4.36/include --oldincludedir=/nix/store/hmxnjscrn38r6lq91h05vc7vpa9pwqdy-libxcrypt-x86_64-unknown-linux-gnu-4.4.36/include --mandir=/nix/store/mx9vjvq5nvk5xq5fccw9v2bpnncdvswf-libxcrypt-x86_64-unknown-linux-gnu-4.4.36-man/share/man --infodir=/nix/store/hmxnjscrn38r6lq91h05vc7vpa9pwqdy-libxcrypt-x86_64-unknown-linux-gnu-4.4.36/share/info --docdir=/nix/store/hmxnjscrn38r6lq91h05vc7vpa9pwqdy-libxcrypt-x86_64-unknown-linux-gnu-4.4.36/share/doc/libxcrypt --libdir=/nix/store/hmxnjscrn38r6lq91h05vc7vpa9pwqdy-libxcrypt-x86_64-unknown-linux-gnu-4.4.36/lib --libexecdir=/nix/store/hmxnjscrn38r6lq91h05vc7vpa9pwqdy-libxcrypt-x86_64-unknown-linux-gnu-4.4.36/libexec --localedir=/nix/store/hmxnjscrn38r6lq91h05vc7vpa9pwqdy-libxcrypt-x86_64-unknown-linux-gnu-4.4.36/share/locale --enable-hashes=strong --enable-obsolete-api=glibc --disable-failure-tokens --disable-werror --build=x86_64-unknown-linux-gnu --host=x86_64-unknown-linux-gnu
checking for a BSD-compatible install... /nix/store/43dnc4h7x0p8ybaq38aaxkhy722kzrx2-coreutils-9.4/bin/install -c
checking whether build environment is sane... yes
checking for a race-free mkdir -p... /nix/store/43dnc4h7x0p8ybaq38aaxkhy722kzrx2-coreutils-9.4/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking for x86_64-unknown-linux-gnu-gcc... x86_64-unknown-linux-gnu-clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether x86_64-unknown-linux-gnu-clang accepts -g... yes
checking for x86_64-unknown-linux-gnu-clang option to enable C11 features... none needed
checking whether x86_64-unknown-linux-gnu-clang understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of x86_64-unknown-linux-gnu-clang... none
checking for x86_64-unknown-linux-gnu-pkg-config... no
checking for pkg-config... no
./configure: line 4840: --atleast-pkgconfig-version: command not found
checking how to run the C preprocessor... x86_64-unknown-linux-gnu-clang -E
checking whether make sets $(MAKE)... (cached) yes
checking whether ln -s works... yes
checking for perl... /nix/store/plxfbnzbqjfn987zl5vrd1pv9qjmjsd3-perl-5.38.2/bin/perl
checking whether /nix/store/plxfbnzbqjfn987zl5vrd1pv9qjmjsd3-perl-5.38.2/bin/perl is version 5.14.0 or later... yes
checking for gpg2... false
checking for sha256sum... /nix/store/43dnc4h7x0p8ybaq38aaxkhy722kzrx2-coreutils-9.4/bin/sha256sum
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for wchar.h... yes
checking for minix/config.h... no
checking for fcntl.h... yes
checking for stdbool.h... yes
checking for ucontext.h... yes
checking for sys/cdefs.h... yes
checking for sys/random.h... yes
checking for sys/syscall.h... yes
checking for valgrind/valgrind.h... no
checking for endian.h... yes
checking for sys/endian.h... no
checking for sys/param.h... yes
checking whether it is safe to define __EXTENSIONS__... yes
checking whether _XOPEN_SOURCE should be defined... no
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking whether C compiler accepts -Werror=unknown-warning-option... yes
checking whether C compiler accepts -Wall... yes
checking whether C compiler accepts -Wextra... yes
checking whether C compiler accepts -Walloc-zero... no
checking whether C compiler accepts -Walloca... yes
checking whether C compiler accepts -Wbad-function-cast... yes
checking whether C compiler accepts -Wcast-align... yes
checking whether C compiler accepts -Wcast-qual... yes
checking whether C compiler accepts -Wconversion... yes
checking whether C compiler accepts -Wformat=2... yes
checking whether C compiler accepts -Wformat-overflow=2... no
checking whether C compiler accepts -Wformat-signedness... no
checking whether C compiler accepts -Wformat-truncation=1... no
checking whether C compiler accepts -Wlogical-op... no
checking whether C compiler accepts -Wmissing-declarations... yes
checking whether C compiler accepts -Wmissing-prototypes... yes
checking whether C compiler accepts -Wnested-externs... yes
checking whether C compiler accepts -Wnull-dereference... yes
checking whether C compiler accepts -Wold-style-definition... yes
checking whether C compiler accepts -Wpointer-arith... yes
checking whether C compiler accepts -Wrestrict... no
checking whether C compiler accepts -Wshadow... yes
checking whether C compiler accepts -Wstrict-overflow=2... yes
checking whether C compiler accepts -Wstrict-prototypes... yes
checking whether C compiler accepts -Wundef... yes
checking whether C compiler accepts -Wvla... yes
checking whether C compiler accepts -Wwrite-strings... yes
checking whether C compiler accepts -Wpedantic... yes
checking how to print strings... printf
checking for a sed that does not truncate output... /nix/store/kqb7yncgd6a17nlgbixhcvcwync0nqg0-gnused-4.9/bin/sed
checking for grep that handles long lines and -e... /nix/store/nk45h8dy9zlx4d0b9ghf5xnj9pypyqgh-gnugrep-3.11/bin/grep
checking for egrep... /nix/store/nk45h8dy9zlx4d0b9ghf5xnj9pypyqgh-gnugrep-3.11/bin/grep -E
checking for fgrep... /nix/store/nk45h8dy9zlx4d0b9ghf5xnj9pypyqgh-gnugrep-3.11/bin/grep -F
checking for ld used by x86_64-unknown-linux-gnu-clang... x86_64-unknown-linux-gnu-ld
checking if the linker (x86_64-unknown-linux-gnu-ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... x86_64-unknown-linux-gnu-nm
checking the name lister (x86_64-unknown-linux-gnu-nm) interface... BSD nm
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-unknown-linux-gnu file names to x86_64-unknown-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-unknown-linux-gnu file names to toolchain format... func_convert_file_noop
checking for x86_64-unknown-linux-gnu-ld option to reload object files... -r
checking for x86_64-unknown-linux-gnu-file... no
checking for file... file
checking for x86_64-unknown-linux-gnu-objdump... x86_64-unknown-linux-gnu-objdump
checking how to recognize dependent libraries... (cached) pass_all
checking for x86_64-unknown-linux-gnu-dlltool... x86_64-unknown-linux-gnu-dlltool
checking how to associate runtime and link libraries... printf %s\n
checking for x86_64-unknown-linux-gnu-ar... x86_64-unknown-linux-gnu-ar
checking for archiver @FILE support... @
checking for x86_64-unknown-linux-gnu-strip... x86_64-unknown-linux-gnu-strip
checking for x86_64-unknown-linux-gnu-ranlib... x86_64-unknown-linux-gnu-ranlib
checking command to parse x86_64-unknown-linux-gnu-nm output from x86_64-unknown-linux-gnu-clang object... ok
checking for sysroot... no
checking for a working dd... /nix/store/43dnc4h7x0p8ybaq38aaxkhy722kzrx2-coreutils-9.4/bin/dd
checking how to truncate binary pipes... /nix/store/43dnc4h7x0p8ybaq38aaxkhy722kzrx2-coreutils-9.4/bin/dd bs=4096 count=1
checking for x86_64-unknown-linux-gnu-mt... no
checking for mt... no
checking if : is a manifest tool... no
checking for dlfcn.h... yes
checking for objdir... .libs
checking if x86_64-unknown-linux-gnu-clang supports -fno-rtti -fno-exceptions... yes
checking for x86_64-unknown-linux-gnu-clang option to produce PIC... -fPIC -DPIC
checking if x86_64-unknown-linux-gnu-clang PIC flag -fPIC -DPIC works... yes
checking if x86_64-unknown-linux-gnu-clang static flag -static works... no
checking if x86_64-unknown-linux-gnu-clang supports -c -o file.o... yes
checking if x86_64-unknown-linux-gnu-clang supports -c -o file.o... (cached) yes
checking whether the x86_64-unknown-linux-gnu-clang linker (x86_64-unknown-linux-gnu-ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking for _ prefix in compiled symbols... no
checking whether the preprocessor (x86_64-unknown-linux-gnu-clang -E) supports -dD... yes
checking whether we are compiling with ASan... no
checking whether sys/cdefs.h defines __BEGIN_DECLS and __END_DECLS... yes
checking whether sys/cdefs.h defines __THROW... yes
checking how to control data alignment... _Alignas
checking how to query data alignment... _Alignof
checking for max_align_t in stddef.h... yes
checking for byte order macros... BYTE_ORDER and xxx_ENDIAN
checking for static_assert in assert.h... yes
checking for ld --wrap... yes
checking linker version script flag... --version-script
checking if version scripts can use complex wildcards... yes
checking for __attribute__((symver))... no
checking how to make linking fail when undefined symbols remain... -Wl,-z,defs
checking how to make linking fail when there are text relocations... -Wl,-z,text
checking how to link with read-only relocations... -Wl,-z,relro
checking how to link with immediate binding... -Wl,-z,now
checking whether C compiler accepts -fno-plt... yes
checking for arc4random_buf... yes
checking for explicit_bzero... yes
checking for explicit_memset... no
checking for getentropy... yes
checking for getrandom... yes
checking for memset_s... no
checking for open64... yes
checking for syscall... yes
checking for valgrind... no
checking for Python 3.>=6 with Passlib... not found
configure: Disabling the "regen-ka-table" target, missing Python requirements.
checking whether all ucontext.h functions are available... no
configure: WARNING: --enable-hashes=strong forces --enable-obsolete-api=no
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating libxcrypt.pc
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands
@nix { "action": "setPhase", "phase": "buildPhase" }
Running phase: buildPhase
build flags: -j8 SHELL=/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash
LC_ALL=C /nix/store/plxfbnzbqjfn987zl5vrd1pv9qjmjsd3-perl-5.38.2/bin/perl \
  ./build-aux/scripts/gen-crypt-hashes-h \
  ./lib/hashes.conf ,bcrypt,bcrypt_a,bcrypt_y,gost_yescrypt,scrypt,sha512crypt,yescrypt, > crypt-hashes.h.T
LC_ALL=C /nix/store/plxfbnzbqjfn987zl5vrd1pv9qjmjsd3-perl-5.38.2/bin/perl \
  ./build-aux/scripts/gen-crypt-symbol-vers-h \
  yes \
  SYMVER_MIN=XCRYPT_2.0 \
  SYMVER_FLOOR=XCRYPT_2.0 \
  COMPAT_ABI=no \
     ./lib/libcrypt.map.in > crypt-symbol-vers.h.T
LC_ALL=C /nix/store/plxfbnzbqjfn987zl5vrd1pv9qjmjsd3-perl-5.38.2/bin/perl \
  ./build-aux/scripts/gen-crypt-h \
  ./lib/crypt.h.in config.h \
  ./lib/hashes.conf ,bcrypt,bcrypt_a,bcrypt_y,gost_yescrypt,scrypt,sha512crypt,yescrypt, \
  > crypt.h.T
./build-aux/scripts/move-if-change crypt-symbol-vers.h.T crypt-symbol-vers.h
echo timestamp > crypt-symbol-vers.h.stamp
./build-aux/scripts/move-if-change crypt-hashes.h.T crypt-hashes.h
./build-aux/scripts/move-if-change crypt.h.T crypt.h
echo timestamp > crypt-hashes.h.stamp
echo timestamp > crypt.h.stamp
LC_ALL=C echo "# Deps" > Makefile.deps.T
mv -f Makefile.deps.T Makefile.deps
make  all-am
make[1]: Entering directory '/build/libxcrypt-4.4.36'
LC_ALL=C /nix/store/plxfbnzbqjfn987zl5vrd1pv9qjmjsd3-perl-5.38.2/bin/perl ./build-aux/scripts/gen-libcrypt-map \
  SYMVER_MIN=XCRYPT_2.0 \
  SYMVER_FLOOR=XCRYPT_2.0 \
  COMPAT_ABI=no \
  ./lib/libcrypt.map.in > libcrypt.map.T
x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/gen-des-tables.o lib/gen-des-tables.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-alg-des-tables.lo `test -f 'lib/alg-des-tables.c' || echo './'`lib/alg-des-tables.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-alg-des.lo `test -f 'lib/alg-des.c' || echo './'`lib/alg-des.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-alg-gost3411-2012-core.lo `test -f 'lib/alg-gost3411-2012-core.c' || echo './'`lib/alg-gost3411-2012-core.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-alg-gost3411-2012-hmac.lo `test -f 'lib/alg-gost3411-2012-hmac.c' || echo './'`lib/alg-gost3411-2012-hmac.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-alg-hmac-sha1.lo `test -f 'lib/alg-hmac-sha1.c' || echo './'`lib/alg-hmac-sha1.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-alg-md4.lo `test -f 'lib/alg-md4.c' || echo './'`lib/alg-md4.c
./build-aux/scripts/move-if-change libcrypt.map.T libcrypt.map
echo timestamp > libcrypt.map.stamp
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-alg-md5.lo `test -f 'lib/alg-md5.c' || echo './'`lib/alg-md5.c
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/alg-md4.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-alg-md4.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/alg-gost3411-2012-hmac.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-alg-gost3411-2012-hmac.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/alg-des-tables.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-alg-des-tables.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/alg-gost3411-2012-core.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-alg-gost3411-2012-core.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/alg-des.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-alg-des.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/alg-hmac-sha1.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-alg-hmac-sha1.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/alg-md5.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-alg-md5.o
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-alg-sha1.lo `test -f 'lib/alg-sha1.c' || echo './'`lib/alg-sha1.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-alg-sha256.lo `test -f 'lib/alg-sha256.c' || echo './'`lib/alg-sha256.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-alg-sha512.lo `test -f 'lib/alg-sha512.c' || echo './'`lib/alg-sha512.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-alg-yescrypt-common.lo `test -f 'lib/alg-yescrypt-common.c' || echo './'`lib/alg-yescrypt-common.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-alg-yescrypt-opt.lo `test -f 'lib/alg-yescrypt-opt.c' || echo './'`lib/alg-yescrypt-opt.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-crypt-bcrypt.lo `test -f 'lib/crypt-bcrypt.c' || echo './'`lib/crypt-bcrypt.c
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/alg-sha1.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-alg-sha1.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/alg-sha512.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-alg-sha512.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/alg-sha256.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-alg-sha256.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/alg-yescrypt-common.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-alg-yescrypt-common.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/alg-yescrypt-opt.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-alg-yescrypt-opt.o
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-crypt-des.lo `test -f 'lib/crypt-des.c' || echo './'`lib/crypt-des.c
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/crypt-bcrypt.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-crypt-bcrypt.o
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-crypt-gensalt-static.lo `test -f 'lib/crypt-gensalt-static.c' || echo './'`lib/crypt-gensalt-static.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-crypt-gost-yescrypt.lo `test -f 'lib/crypt-gost-yescrypt.c' || echo './'`lib/crypt-gost-yescrypt.c
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/crypt-des.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-crypt-des.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/crypt-gensalt-static.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-crypt-gensalt-static.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/crypt-gost-yescrypt.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-crypt-gost-yescrypt.o
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-crypt-md5.lo `test -f 'lib/crypt-md5.c' || echo './'`lib/crypt-md5.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-crypt-nthash.lo `test -f 'lib/crypt-nthash.c' || echo './'`lib/crypt-nthash.c
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/crypt-md5.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-crypt-md5.o
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-crypt-pbkdf1-sha1.lo `test -f 'lib/crypt-pbkdf1-sha1.c' || echo './'`lib/crypt-pbkdf1-sha1.c
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/crypt-nthash.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-crypt-nthash.o
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-crypt-scrypt.lo `test -f 'lib/crypt-scrypt.c' || echo './'`lib/crypt-scrypt.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-crypt-sha256.lo `test -f 'lib/crypt-sha256.c' || echo './'`lib/crypt-sha256.c
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/crypt-pbkdf1-sha1.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-crypt-pbkdf1-sha1.o
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-crypt-sha512.lo `test -f 'lib/crypt-sha512.c' || echo './'`lib/crypt-sha512.c
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/crypt-scrypt.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-crypt-scrypt.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/crypt-sha256.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-crypt-sha256.o
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-crypt-static.lo `test -f 'lib/crypt-static.c' || echo './'`lib/crypt-static.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-crypt-sunmd5.lo `test -f 'lib/crypt-sunmd5.c' || echo './'`lib/crypt-sunmd5.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-crypt-yescrypt.lo `test -f 'lib/crypt-yescrypt.c' || echo './'`lib/crypt-yescrypt.c
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/crypt-sha512.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-crypt-sha512.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/crypt-sunmd5.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-crypt-sunmd5.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/crypt-static.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-crypt-static.o
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-crypt.lo `test -f 'lib/crypt.c' || echo './'`lib/crypt.c
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/crypt-yescrypt.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-crypt-yescrypt.o
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-util-base64.lo `test -f 'lib/util-base64.c' || echo './'`lib/util-base64.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-util-gensalt-sha.lo `test -f 'lib/util-gensalt-sha.c' || echo './'`lib/util-gensalt-sha.c
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/util-base64.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-util-base64.o
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-util-get-random-bytes.lo `test -f 'lib/util-get-random-bytes.c' || echo './'`lib/util-get-random-bytes.c
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/crypt.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-crypt.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/util-gensalt-sha.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-util-gensalt-sha.o
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-util-make-failure-token.lo `test -f 'lib/util-make-failure-token.c' || echo './'`lib/util-make-failure-token.c
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/util-get-random-bytes.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-util-get-random-bytes.o
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-util-xbzero.lo `test -f 'lib/util-xbzero.c' || echo './'`lib/util-xbzero.c
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I.  -I./lib -DIN_LIBCRYPT  -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c -o lib/libcrypt_la-util-xstrcpy.lo `test -f 'lib/util-xstrcpy.c' || echo './'`lib/util-xstrcpy.c
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/util-make-failure-token.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-util-make-failure-token.o
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=link x86_64-unknown-linux-gnu-clang -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -Wl,-z,relro -Wl,-z,now  -o lib/gen-des-tables lib/gen-des-tables.o  
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/util-xbzero.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-util-xbzero.o
libtool: compile:  x86_64-unknown-linux-gnu-clang -DHAVE_CONFIG_H -I. -I./lib -DIN_LIBCRYPT -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -c lib/util-xstrcpy.c  -fPIC -DPIC -o lib/.libs/libcrypt_la-util-xstrcpy.o
libtool: link: x86_64-unknown-linux-gnu-clang -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -Wl,-z -Wl,relro -Wl,-z -Wl,now -o lib/gen-des-tables lib/gen-des-tables.o 
/nix/store/5lr5n3qa4day8l1ivbwlcby2nknczqkq-bash-5.2p26/bin/bash ./libtool  --tag=CC   --mode=link x86_64-unknown-linux-gnu-clang -Wall -Wextra -Walloca -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wnull-dereference -Wold-style-definition -Wpointer-arith -Wshadow -Wstrict-overflow=2 -Wstrict-prototypes -Wundef -Wvla -Wwrite-strings -Wpedantic -fno-plt -g -O2 -version-info 2:0:0 -Wl,--version-script,./libcrypt.map -Wl,-z,defs -Wl,-z,text -Wl,-z,relro -Wl,-z,now  -o libcrypt.la -rpath /nix/store/hmxnjscrn38r6lq91h05vc7vpa9pwqdy-libxcrypt-x86_64-unknown-linux-gnu-4.4.36/lib lib/libcrypt_la-alg-des-tables.lo lib/libcrypt_la-alg-des.lo lib/libcrypt_la-alg-gost3411-2012-core.lo lib/libcrypt_la-alg-gost3411-2012-hmac.lo lib/libcrypt_la-alg-hmac-sha1.lo lib/libcrypt_la-alg-md4.lo lib/libcrypt_la-alg-md5.lo lib/libcrypt_la-alg-sha1.lo lib/libcrypt_la-alg-sha256.lo lib/libcrypt_la-alg-sha512.lo lib/libcrypt_la-alg-yescrypt-common.lo lib/libcrypt_la-alg-yescrypt-opt.lo lib/libcrypt_la-crypt-bcrypt.lo lib/libcrypt_la-crypt-des.lo lib/libcrypt_la-crypt-gensalt-static.lo lib/libcrypt_la-crypt-gost-yescrypt.lo lib/libcrypt_la-crypt-md5.lo lib/libcrypt_la-crypt-nthash.lo lib/libcrypt_la-crypt-pbkdf1-sha1.lo lib/libcrypt_la-crypt-scrypt.lo lib/libcrypt_la-crypt-sha256.lo lib/libcrypt_la-crypt-sha512.lo lib/libcrypt_la-crypt-static.lo lib/libcrypt_la-crypt-sunmd5.lo lib/libcrypt_la-crypt-yescrypt.lo lib/libcrypt_la-crypt.lo lib/libcrypt_la-util-base64.lo lib/libcrypt_la-util-gensalt-sha.lo lib/libcrypt_la-util-get-random-bytes.lo lib/libcrypt_la-util-make-failure-token.lo lib/libcrypt_la-util-xbzero.lo lib/libcrypt_la-util-xstrcpy.lo   
libtool: link: x86_64-unknown-linux-gnu-clang -shared  -fPIC -DPIC  lib/.libs/libcrypt_la-alg-des-tables.o lib/.libs/libcrypt_la-alg-des.o lib/.libs/libcrypt_la-alg-gost3411-2012-core.o lib/.libs/libcrypt_la-alg-gost3411-2012-hmac.o lib/.libs/libcrypt_la-alg-hmac-sha1.o lib/.libs/libcrypt_la-alg-md4.o lib/.libs/libcrypt_la-alg-md5.o lib/.libs/libcrypt_la-alg-sha1.o lib/.libs/libcrypt_la-alg-sha256.o lib/.libs/libcrypt_la-alg-sha512.o lib/.libs/libcrypt_la-alg-yescrypt-common.o lib/.libs/libcrypt_la-alg-yescrypt-opt.o lib/.libs/libcrypt_la-crypt-bcrypt.o lib/.libs/libcrypt_la-crypt-des.o lib/.libs/libcrypt_la-crypt-gensalt-static.o lib/.libs/libcrypt_la-crypt-gost-yescrypt.o lib/.libs/libcrypt_la-crypt-md5.o lib/.libs/libcrypt_la-crypt-nthash.o lib/.libs/libcrypt_la-crypt-pbkdf1-sha1.o lib/.libs/libcrypt_la-crypt-scrypt.o lib/.libs/libcrypt_la-crypt-sha256.o lib/.libs/libcrypt_la-crypt-sha512.o lib/.libs/libcrypt_la-crypt-static.o lib/.libs/libcrypt_la-crypt-sunmd5.o lib/.libs/libcrypt_la-crypt-yescrypt.o lib/.libs/libcrypt_la-crypt.o lib/.libs/libcrypt_la-util-base64.o lib/.libs/libcrypt_la-util-gensalt-sha.o lib/.libs/libcrypt_la-util-get-random-bytes.o lib/.libs/libcrypt_la-util-make-failure-token.o lib/.libs/libcrypt_la-util-xbzero.o lib/.libs/libcrypt_la-util-xstrcpy.o    -g -O2 -Wl,--version-script -Wl,./libcrypt.map -Wl,-z -Wl,defs -Wl,-z -Wl,text -Wl,-z -Wl,relro -Wl,-z -Wl,now   -Wl,-soname -Wl,libcrypt.so.2 -o .libs/libcrypt.so.2.0.0
x86_64-unknown-linux-gnu-ld: error: version script assignment of 'XCRYPT_2.0' to symbol 'crypt_gensalt_r' failed: symbol not defined
x86_64-unknown-linux-gnu-ld: error: version script assignment of 'XCRYPT_2.0' to symbol 'xcrypt' failed: symbol not defined
x86_64-unknown-linux-gnu-ld: error: version script assignment of 'XCRYPT_2.0' to symbol 'xcrypt_gensalt' failed: symbol not defined
x86_64-unknown-linux-gnu-ld: error: version script assignment of 'XCRYPT_2.0' to symbol 'xcrypt_gensalt_r' failed: symbol not defined
x86_64-unknown-linux-gnu-ld: error: version script assignment of 'XCRYPT_2.0' to symbol 'xcrypt_r' failed: symbol not defined
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [Makefile:1530: libcrypt.la] Error 1
make[1]: Leaving directory '/build/libxcrypt-4.4.36'
make: *** [Makefile:1349: all] Error 2
```

### Notify maintainers

@dottedmag 
@hexa

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.7.9-xanmod1, NixOS, 24.05 (Uakari), 24.05.20240312.0ad13a6`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - nixpkgs: `/nix/store/k5l01g2zwhysjyl5zjvg5zxnj0lyxpp1-source`
```

--- a/pkgs/development/libraries/libxcrypt/default.nix
+++ b/pkgs/development/libraries/libxcrypt/default.nix
@@ -28,8 +28,18 @@ stdenv.mkDerivation (finalAttrs: {
     "--disable-werror"
   ];
 
-  # fixes: can't build x86_64-w64-mingw32 shared library unless -no-undefined is specified
-  makeFlags = lib.optionals stdenv.hostPlatform.isWindows [ "LDFLAGS=-no-undefined"] ;
+  makeFlags = let
+    lld17Plus = stdenv.cc.bintools.isLLVM
+      && lib.versionAtLeast stdenv.cc.bintools.version "17";
+  in []
+    # fixes: can't build x86_64-w64-mingw32 shared library unless -no-undefined is specified
+    ++ lib.optionals stdenv.hostPlatform.isWindows [ "LDFLAGS+=-no-undefined" ]
+
+    # lld 17 sets `--no-undefined-version` by default and `libxcrypt`'s
+    # version script unconditionally lists legacy compatibility symbols, even
+    # when not exported: https://github.com/besser82/libxcrypt/issues/181
+    ++ lib.optionals lld17Plus [ "LDFLAGS+=-Wl,--undefined-version" ]
+  ;
 
   nativeBuildInputs = [
     perl

